lib/common/parse-features-string.js

```diff
@@ -2,6 +2,7 @@
 // - `features` input string
 // - `emit` function(key, value) - called for each parsed KV
 module.exports = function parseFeaturesString (features, emit) {
+  features  = String(features)
```

Comment:
We have two ways of doing this string casting in our code.  In some places we do it like this and in others we use 

```js
features = `${features}`
```

I'm not sure it matters but should we stick with one over the other? 😆